index.html

```diff
@@ -0,0 +1,480 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Technical Document</title>
+    <link rel="stylesheet" href="./styles.css"
+  </head>
+  <body>
+    <nav id="navbar">
+      <header>JS Documentation</header>
```

Comment:
Please header element  should enclose nav element and not the reverse for html practice

Reply:
> Please header element should enclose nav element and not the reverse for HTML practice

- [ ] As indicated in the comment above, also indicate the reason for enclosing the header element with `<nav>` tag.